lib/ansible/modules/system/pids.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: pids
+version_added: 2.8
+description: "Retrieves a list of process IDs (PIDs) of all processes of the given name. Returns an empty list if no process of the given name exists."
+short_description: "Retrieves process IDs list if the process is running otherwise return empty list"
+author:
+  - Saranya Sridharan (@saranyasridharan)
+requirements:
+  - Needs 'pidof' to be installed in the system
+options:
+  name:
+    description: the name of the process you want to get PID for
+    required: true
+'''
+EXAMPLES = '''
+# Pass the process name
+- name: Getting process IDs of the process
+  pids:
+      name: python
+  register: pids_of_python
+
+- name: Printing the process IDs obtained
+  debug: msg="PIDS of python:{{','.join(pids_of_python.pid)}}"
+'''
+
+RETURN = '''
+pids:
+  description: It return process IDs of the given process
+  returned: list of none, one, or more process IDs
+  type: list
+'''
+
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def get_pid(name,module):
+    command = module.get_bin_path('pidof', True)
+	command+ = name
```

Comment:
You have a lot of spacing problems in the above three lines. Please install `flake8` on your machine (with `pip install flake8`) and run it for this module (`flake8 pids.py`), that will show you a lot of the errors much earlier.

Also, `command` is a string, not an array, so you either need to write `command = [command, name]`, or you do `module.run_command([command, name])` in the next line.